libsolidity/ast/AST_accept.h

```diff
@@ -239,6 +239,28 @@ void VariableDeclaration::accept(ASTConstVisitor& _visitor) const
 	_visitor.endVisit(*this);
 }
 
+void ModifierArea::accept(ASTVisitor& _visitor)
+{
+	if (_visitor.visit(*this))
+	{
+		listAccept(m_declaredModifiers, _visitor);
+		listAccept(*m_functions, _visitor);
```

Comment:
Somewhere you should probably assert that these members (``m_functions``, ``m_subAreas``) aren't null pointers. Either here or in the constructor of ``ModifierArea`` or even at both places (or otherwise this should be an ``if (m_functions) listAccept(...)`` if the pointer may actually be null, although it doesn't look like they can... as stated above it may make sense for them not to be pointer at all).